apps/server/src/modules/tool/school-external-tool/service/school-external-tool-authorizable.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain/types';
+import { SchoolExternalToolRepo } from '@shared/repo';
+import {
+	AuthorizableReferenceType,
+	AuthorizationInjectionService,
+	AuthorizationLoaderServiceGeneric,
+} from '@src/modules/authorization';
+import { SchoolExternalTool } from '../domain';
+
+@Injectable()
+export class SchoolExternalToolAuthorizableService implements AuthorizationLoaderServiceGeneric<SchoolExternalTool> {
```

Comment:
AuthorizationLoaderServiceGeneric sollte eigentlich auch bereits entfernt werden und durch AuthorizationLoaderService ersetzt werden. Ich sehe erstmal kein Grund das sich bei Tools nicht direkt verwenden lässt. Da die Domain Objekte alle von AuthorizableObject extenden.

Hier wäre schön wenn du noch mal schauen könntest, ob sich das direkt überall in Tools austauschen lässt. 

Edit: Eigentlich sogar in allen Bereichen das ist ja jetzt sogar in group, instance und course-do.service gelandet...

Evtl.  aber eine Arbeit für ein extra Ticket, sicher schnell gemacht. Aber anderer Fokus. 
Todo: Ticket erstellen, wer erstellt es?